### PR TITLE
 Add cypress test to profile

### PR DESCRIPTION
**Pull request by lynchl607**
Add cypress test to profile
Add vault functionality

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,8 +15,10 @@
   "devDependencies": {
     "@stencil/core": "^1.12.2",
     "@stencil/router": "^1.0.1",
-    "stencil-inline-svg": "^1.0.1",
-    "crds-components-env-replace": "github:crdschurch/crds-components-env-replace#1.1.1"
+    "crds-components-env-replace": "github:crdschurch/crds-components-env-replace#1.1.1",
+    "crds-cypress-config": "^1.0.1",
+    "cypress": "^5.2.0",
```
Comment (s-henney) [inline package.json:20]: Could you add a script to the 'scripts' section that'll kick off Cypress? I normally include one that runs in headless mode with the int_crossroads.json config file.

**Review by s-henney — CHANGES_REQUESTED** [review]
You've added good coverage here, nice job!
You should alias and reuse elements to make tests more readable though. I included an example in the Contact Login Tab suite, and you can checkout Cypress's examples [here](https://docs.cypress.io/api/commands/get.html#Alias).

File: cypress/integration/profille/contact_and_login_tab_spec.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    cy.get('crds-phone'  , { includeShadowDom: true }).get('[placeholder="Mobile Number"]', { includeShadowDom: true }).click({ force: true }).clear({ force: true });
+
+    cy.get('crds-phone'  , { includeShadowDom: true }).get('[placeholder="Mobile Number"]', { includeShadowDom: true }).get('p' , { includeShadowDom: true }, { timeout: 15000 }).contains('At least one phone number is required.')
+
+    cy.get('crds-phone'  , { includeShadowDom: true }).get('[placeholder="Home Number"]', { includeShadowDom: true }).clear({ force: true });
+
+    cy.get('crds-phone'  , { includeShadowDom: true }).get('[placeholder="Home Number"]', { includeShadowDom: true }).get('p' , { includeShadowDom: true }, { timeout: 15000 }).contains('At least one phone number is required.')
+
+    cy.get('crds-phone'  , { includeShadowDom: true }).get('[placeholder="Mobile Number"]', { includeShadowDom: true }).click();
+    cy.get('crds-phone'  , { includeShadowDom: true }).get('[placeholder="Mobile Number"]', { includeShadowDom: true }).type('555', { force: true });
```
[the remark this answers is not in the document]
Comment (lynchl607) [reply cypress/integration/profille/contact_and_login_tab_spec.js:53]: The  { force: true } is necessary with out it I get a disabled element error.

**Review by s-henney — CHANGES_REQUESTED** [review]
I like how you've aliased the elements in the `beforeEach` blocks - makes the tests nice and easy to read.
In many `before` blocks you're navigating the the `/profile/personal` page, waiting on the redirect, then clicking the link in the side panel to get to the page you want. Don't do this. You already have a test that confirms the redirect works, and that each link navigates to the correct page, so running those same steps in each test file just takes extra time but adds no value. Just navigate directly to the page you want to test and start testing.

**Review by s-henney — APPROVED** [review]
Looks good!

**Pull request merged**